frontend/public/components/RBAC/role.jsx

```diff
@@ -33,9 +33,12 @@ const menuActions = [
   (kind, role) => ({
     label: 'Add Role Binding',
     href: `/k8s/${
-      role.metadata.namespace ? `ns/${role.metadata.namespace}` : 'cluster'
-    }/rolebindings/~new?rolekind=${roleKind(role)}&rolename=${role.metadata.name}${role.metadata
-      .namespace && `&namespace=${role.metadata.namespace}`}`,
+      role.metadata.namespace
+        ? `ns/${role.metadata.namespace}/rolebindings/~new?rolekind=${roleKind(role)}&rolename=${
+            role.metadata.name
+          }${role.metadata.namespace && `&namespace=${role.metadata.namespace}`}`
```

Comment:
We already check for `role.metadata.namespace` just above, so we shouldn't need to check again.

Reply:
@spadgett Updated, thanks.